lib/vanagon/platform/windows.rb

```diff
@@ -57,6 +57,16 @@ def generate_msi_packaging_artifacts(workdir, name, binding)
         erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/project.wxs.erb"), File.join(workdir, "wix",  "#{name}.wxs"), false, { :binding => binding })
         erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/service.component.wxs.erb"), File.join(workdir, "wix", "service.#{name}.wxs"), false, { :binding => binding })
         erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/project.filter.xslt.erb"), File.join(workdir, "wix", "#{name}.filter.xslt"), false, { :binding => binding })
+
+        # Project Specific Wix Files (ERB Templates)
+        ["customactions","properties","sequences"].each do |wix_filename|
```

Comment:
Not necessarily.

Some custom actions may be generic, so we could have a set of generic ones in Vanagon that are deemed useful across multiple projects, with the more specific ones sitting in the project area.
You can also specify as many "fragment" files containing custom actions as you want.

The same applies to property files - these can be specified in both generic and project specific area.

The Sequences however, will be project specific (<ExecuteSequence, UISequence).